package.json

```diff
@@ -63,7 +63,6 @@
 		"@testing-library/react-hooks": "npm:@testing-library/preact-hooks",
 		"@types/compression": "^1.7.5",
 		"@types/cookie-parser": "^1.4.8",
-		"@types/eslint__js": "^8.42.3",
```

Comment:
Also removed this as `knip` (`pnpm run knip`) suggested that it's not used anywhere